trackintel/preprocessing/positionfixes.py

```diff
@@ -244,6 +245,13 @@ def generate_triplegs(
             # initialize the index list of pfs where a tpl will begin
             insert_index_ls = []
             pfs["staypoint_id"] = pd.NA
+
+            # check if print_progress is True.
+            if print_progress:
```

Comment:
I think the tqdm update bar can be directly incorporated into line 255 (the for loop) with the argument `disable`. Check the example [here](https://github.com/mie-lab/trackintel/blob/5c4ac2afce498207486bf6e03fac4a81ad8c6b99/trackintel/io/dataset_reader.py#L183) and [here](https://github.com/mie-lab/trackintel/blob/5c4ac2afce498207486bf6e03fac4a81ad8c6b99/trackintel/preprocessing/util.py#L82).